Prog/PresStudyTabulations.sas

```diff
@@ -0,0 +1,369 @@
+
+%let rpt_suffix = %sysfunc( putn( %sysfunc( today() ), yymmddn8. ) );
+
+options orientation=landscape;
+
+options missing='0';
+options nodate nonumber;
+
+%fdate()
+
+ods rtf file="&_dcdata_default_path\PresCat\Prog\Project_assisted_units_&rpt_suffix..rtf" style=Styles.Rtf_arial_9pt;
+
+*****************Study Tabulations below here*********
+************Tables by Location and Subsidy*********;
+
+title3 "Project and assisted unit unique counts by ward";
+
+proc tabulate data=Project_assisted_units format=comma10. noseps missing;
+  where ProgCat ~= . and not( missing( ward2022 ) );
+  class ProgCat / preloadfmt order=data;
+  class ward2022;
+  var mid_asst_units err_asst_units;
+  table 
+    /** Rows **/
+    ( all='DC Total' ward2022=' ' )
+    ,
+    /** Columns **/
+    n='Projects' * ( all='\b Total' ProgCat=' ' ) * mid_asst_units=' '
+    ;
+  table 
+    /** Rows **/
+    ( all='DC Total' ward2022=' ' )
+    ,
+    /** Columns **/
+    sum='Assisted Units' * ( all='\b Total' ProgCat=' ' ) * mid_asst_units=' '
+    ;
+  format ProgCat ProgCat.;
+run;
+
+title3 "Project and assisted unit unique counts by ANC";
+
+proc tabulate data=Project_assisted_units format=comma10. noseps missing;
+  where ProgCat ~= . and not( missing( anc2012 ) );
+  class ProgCat / preloadfmt order=data;
+  class anc2012;
+  var mid_asst_units err_asst_units;
+  table 
+    /** Rows **/
+    ( all='DC Total' anc2012=' ' )
+    ,
+    /** Columns **/
+    n='Projects' * ( all='\b Total' ProgCat=' ' ) * mid_asst_units=' '
+    ;
+  table 
+    /** Rows **/
+    ( all='DC Total' anc2012=' ' )
+    ,
+    /** Columns **/
+    sum='Assisted Units' * ( all='\b Total' ProgCat=' ' ) * mid_asst_units=' '
+    ;
+  format ProgCat ProgCat.;
+run;
+
+title3 "Project and assisted unit unique counts by Neighborhood Custer";
+
+proc tabulate data=Project_assisted_units format=comma10. noseps missing;
+  where ProgCat ~= . and not( missing( cluster2017 ) );
+  class ProgCat / preloadfmt order=data;
+  class cluster2017;
+  var mid_asst_units err_asst_units;
+  table 
+    /** Rows **/
+    ( all='DC Total' cluster2017=' ' )
+    ,
+    /** Columns **/
+    n='Projects' * ( all='\b Total' ProgCat=' ' ) * mid_asst_units=' '
+    ;
+  table 
+    /** Rows **/
+    ( all='DC Total' cluster2017=' ' )
+    ,
+    /** Columns **/
+    sum='Assisted Units' * ( all='\b Total' ProgCat=' ' ) * mid_asst_units=' '
+    ;
+  format ProgCat ProgCat.;
+run;
+
+****Creat Data set for Age of Building Table***;
+data Project_AOB;
+
+  merge
+    Project_assisted_units
+		(in=inProject)
+    Prescat.Parcel
+		(in=inParcel);
+  by NLIHC_ID;
+  	if inProject and inParcel;
+   
+run;
+*****Errors Start somewhere around here******;
+
+data Project_Age_Of_Building;
+
+  merge
+   Project_AOB
+   	(in=inProject)
+   Realprop.Cama_parcel_2023_05
+   	(in=inCama);
+
+  by SSL;
+ 	 if inProject and inCama;
+   
+run;
+
+proc summary data=Project_Age_Of_Building 
+  by nlihc_id;
```

Comment:
`ERROR 22-322: Syntax error, expecting one of the following: ;, (, ...`

Missing a semicolon at the end of the Proc Summary statement.